Assembler.Base/BaseMessageHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Assembler.Core;
+using Assembler.Core.Entities;
+
+namespace Assembler.Base
+{
+    public abstract class BaseMessageHandler<TFrame, TMessage> : IHandler<TFrame, TMessage>
+        where TFrame : BaseFrame
+        where TMessage : BaseMessageInAssembly
+    {
+        protected readonly ITimeBasedCache<TMessage> Cache;
+        protected readonly IFactory<TFrame, string> IdentifierFactory;
+        protected readonly IMessageEnricher<TFrame, TMessage> MessageEnricher;
+        protected readonly ICreator<TMessage> AssembledMessageCreator;
```

Comment:
Shouldn't this be `MessageInAssemblyCreator`?

Reply:
f.
Also changed the file names as per you r last comment.

Reply:
👍